test/unit/faker.test.js

```diff
@@ -41,4 +41,21 @@ describe('JSON SCHEMA FAKER TESTS', function () {
       expect(fakedData).to.deep.equal(schema.example);
     });
   });
+
+  it('Should not use actual property named "default" as faked value', function () {
+    const schema = {
+      type: 'object',
+      properties: {
+        default: {
+          type: 'string',
+          example: 'This is actual property and not JSON schema defined "default" keyword'
+        }
+      }
+    };
+
+    var fakedData = schemaFaker(schema);
+    expect(fakedData).to.deep.equal({
```

Comment:
@VShingala what would the value of fakedData have been without the fix?

Reply:
so without fix it would be, entire schema as value

```
{
  "example": "This is actual property and not JSON schema defined \"default\" keyword"
  "type": "string"
}
```

and with fix it will be:
```
{
  "default": "This is actual property and not JSON schema defined \"default\" keyword"
}
```